main.tex

```diff
@@ -332,7 +332,14 @@ \subsection{Updating test files}
 \section{Implementation}
 \label{sec:implementation}
 
-Before Markdown 3.0.0, our framework was implemented by a Bash script \file{test.sh}, see Listing~\ref{lst:test.sh}.
+Before Markdown 3.0.0, our framework was implemented by a Bash script \file{test.sh}, shown in Listing~\ref{lst:test.sh}.
```

Comment:
This change seems unnecessary and produces an extra short line that looks unseemly.

```suggestion
Before Markdown 3.0.0, our framework was implemented by a Bash script \file{test.sh}, see Listing~\ref{lst:test.sh}.
```